desktop/apps/artsy_primer/routes.js

```diff
@@ -19,3 +22,22 @@ export const personalize = async (req, res) => {
   }
   res.render('personalize')
 }
+
+export const setSailthruData = async (req, res) => {
```

Comment:
Nice, so this should be enough to disable them from getting the emails we don't want them to? (ie- can be handled by email/ST team).